cliconfig/credentials/native_store_test.go

```diff
@@ -0,0 +1,170 @@
+// +build test_unit
```

Comment:
I'm using that same tag to load a mock command for testing, to avoid shelling out during tests. I'm still considering if it's the best way to simulate commands. I'm open to suggestions